src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -6284,6 +6284,9 @@ static void figuredBass(XmlWriter& xml, track_idx_t strack, track_idx_t etrack,
             if (track == wtrack) {
                 if (e->type() == ElementType::FIGURED_BASS) {
                     const FiguredBass* fb = dynamic_cast<const FiguredBass*>(e);
+                    if (fb->items().empty()) {
+                        return;
```

Comment:
Actually this should probably be `continue` rather than a `return`... Sorry, it was my fault that I didn't spot that when merging the master version!

Reply:
Whoops. You're absolutely right. I'll provide a fix for master.